core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -21,31 +20,31 @@ import akka.persistence.cassandra.query.scaladsl.CassandraReadJournal
 import akka.persistence.cassandra.session.scaladsl.CassandraSession
 import akka.persistence.journal.{ AsyncWriteJournal, Tagged }
 import akka.persistence.query.PersistenceQuery
-import akka.serialization.{ Serialization, SerializationExtension }
+import akka.serialization.{ AsyncSerializer, Serialization, SerializationExtension }
 import akka.stream.ActorMaterializer
+import akka.util.OptionVal
 import com.datastax.driver.core._
 import com.datastax.driver.core.exceptions.DriverException
 import com.datastax.driver.core.policies.RetryPolicy.RetryDecision
 import com.datastax.driver.core.policies.{ LoggingRetryPolicy, RetryPolicy }
 import com.datastax.driver.core.utils.{ Bytes, UUIDs }
 import com.typesafe.config.Config
-import scala.collection.immutable.Seq
+
 import scala.collection.JavaConverters._
+import scala.collection.immutable
+import scala.collection.immutable.Seq
 import scala.concurrent._
 import scala.util.control.NonFatal
 import scala.util.{ Failure, Success, Try }
 
-import akka.serialization.AsyncSerializer
-import akka.util.OptionVal
-
 class CassandraJournal(cfg: Config) extends AsyncWriteJournal
   with CassandraRecovery
   with CassandraStatements
   with NoSerializationVerificationNeeded {
 
-  private[akka] val config = new CassandraJournalConfig(context.system, cfg)
-  private[akka] val serialization = SerializationExtension(context.system)
-  private[akka] val log: LoggingAdapter = Logging(context.system, getClass)
+  val config = new CassandraJournalConfig(context.system, cfg)
```

Comment:
protected?

Reply:
sure

Reply:
hmm
value config has weaker access privileges; it should be at least be private[akka] since we subclass CassandraRecovery
-> I would like to allow this access but up to you